pkg/osquery/extension.go

```diff
@@ -75,6 +75,9 @@ const (
 	// the bucket which we push values into from server-backed tables
 	ServerProvidedDataBucket = "server_provided_data"
 
+	// the bucket where we hold sent notifications
+	SentNotificationsBucket = "sent_notifications"
```

Comment:
Curious if we intend to expose notification data to osquery, or if this was just a convenient spot to define it?

Reply:
No current plans to expose it. But this is where the bucket name constants live. (Kinda silly really, but :shrug:)